docs/0.1/location_transaction_family.md

```diff
@@ -0,0 +1,280 @@
+[Sawtooth Sabre](https://github.com/hyperledger/sawtooth-sabre/)
+smart contract engine.
+
+Location is designed to supply Hyperledger Grid with a generic and extendable
+framework for storing and managing location entities. Location offers explicit
+support for locations defined using the [GS1 data standard](https://www.gs1.org/),
+but is also designed to be flexible enough to support other standards.
+
+Location uses the Pike smart contract to manage location create, update, and
+delete permissions, as well as the organizations that own each location, and
+uses Grid Hyperledger Schema to describe the requirements for location entities.
+
+## State
+
+### Location
+
+A location is an arbitrary list of properties that is uniquely identified by a
+`location_id`. The `properties` are described by a schema that was previously
+defined using the Hyperledger Grid Schema smart contract, which itself
+corresponds to a `LocationNamespace`. Currently, only the `GS1` namespace has
+been defined.
+
+A location has four fields
+
+* location_id: A unique identifier for the location. For the GS1 namespace the
+  location_id is a Global Location Number or [GLN](https://www.gs1.org/standards/id-keys/gln).
+* namespace: The namespace that the location belongs to. A location's namespace
+  dictates the rules for defining its `location_id` and the properties belonging
+  to the location. Currently only the GS1 namespace is defined.
+* owner: The Pike organization ID of the organization that owns the location.
+* properties: An arbitrary list of properties that describe the location. These properties
+  are defined by a schema defined by the Hyperledger Grid smart contract.
+
+```protobuf
+    message Location {
+        enum LocationNamespace {
+            UNSET_TYPE = 0;
+            GS1 = 1;
+        }
+        // Global Location Number as defined by GS1 specification
+        string location_id = 1;
+
+        LocationNamespace namespace = 2;
+
+        // Who owns this product (pike organization id)
+        string owner = 3;
+
+        // Addition attributes for custom configurations 
+        repeated PropertyValue properties = 4;
+    }
+```
+
+### Location List
+
+Locations whose addresses collide are stored in a location list. A location list
+contains only one field.
+
+```protobuf
+    message LocationList {
+        repeated Location entries = 1;
+    }
+```
+
+### Addressing for GS1 locations
+
+In order to uniquely locate GS1 locations in the Merkle-Radix state system, an
+address must be constructed which identifies the storage location of the
+Location representation.
+
+All Grid addresses are prefixed by the 6-hex-character namespace prefix “621dee”.
+Locations are further prefixed under the Grid namespace with reserved
+enumerations of “04” (“00” = TBD, “01” = Schema, “02” = Product, and “03” =
+Catalog) indicating “Locations” and an additional “01” indicating “GS1
+Locations”.
```

Comment:
The structure of this sentence makes it difficult to tell which prefix indicates "Locations".  Also, the following '`and an additional "01"`' made me think that you add 01 to one of the other prefixes, like "02", to get the GS1 type for any smart contract prefix. 

It might be clearer to move the parenthetical text to the end of the sentence, then describe the GS1 "location standards type" prefix in a separate sentence.  For example: 

> The next two characters in the prefix indicate the smart contract type. “04” indicates the Locations smart contract. (The other enumerations: “00” = TBD, “01” = Schema, “02” = Product, and “03” = Catalog.)  An additional “01” prefix indicates a GS1 location type.

Reply:
Note that you could assume that the other specs document their own prefixes, so the "other" list isn't essential here. But it is helpful to see the full list!  (It would be nice to have this info in a separate reference topic or FAQ entry someday soon.)